src/utils/api/api.js

```diff
@@ -8,6 +8,9 @@ async function api(endpoint, init = {}) {
     case 'roles':
       url = '/jsonapi/user_role/user_role';
       break;
+    case 'content':
+      url = '/jsonapi/node/article';
```

Comment:
Now that https://github.com/jsdrupal/drupal-admin-ui/pull/96 was merged you should be able to change this to `/json/node` to query all node entities.